.bp-config/httpd/extra/httpd-default.conf

```diff
@@ -38,7 +38,7 @@ RequestReadTimeout header=20-40,MinRate=500 body=20,MinRate=500
 
     # Don't compress images
     SetEnvIfNoCase Request_URI \
-    \.(?:gif|jpe?g|png|swf|xsd)$ no-gzip dont-vary
+    \.(?:gif|jpe?g|png|swf|xsd)$|thumbnail-by-url.json$ no-gzip dont-vary
```

Comment:
I think this should be:
```
    (\.(?:gif|jpe?g|png|swf|xsd)|thumbnail-by-url.json)$ no-gzip dont-vary
```

Reply:
I think my initial regex should work, unless there are some restrictions related to grouping I'm not aware of. In any case your version is probably a bit easier to read and matches both types of requests in the first group, so I changed it to your version